src/store/media.js

```diff
@@ -166,16 +158,13 @@ export const createActions = (services = mediaServices) => ({
     const resultRank = Object.keys(state.results[mediaType].items).findIndex(
       (item) => item === id
     )
-    await dispatch(
-      `${USAGE_DATA}/${SEND_RESULT_CLICKED_EVENT}`,
-      {
-        query: rootState.search.query.q,
-        resultUuid: id,
-        resultRank,
-        sessionId: rootState.user.usageSessionId,
-      },
-      { root: true }
-    )
+    const eventData = {
+      query: rootState.search.query.q,
+      resultUuid: id,
+      resultRank,
+      sessionId: rootState.user.usageSessionId,
+    }
+    usageData.sendResultClickedEvent(eventData, this)
```

Comment:
I _really_ dislike how Nuxt plugins lock things like Sentry inside the Nuxt context instead of just being available via an import. It doesn't make any sense for us to have to pass around the context to utility methods :frowning_face:

Reply:
I agree, it feels somehow wrong but still is how have been working til now.